pkg/network/setup/network.go

```diff
@@ -67,14 +68,15 @@ func (v VMNetworkConfigurator) getPhase2NICs(domain *api.Domain, networks []v1.N
 	var nics []podNIC
 
 	for i := range networks {
+		// SR-IOV devices are not part of the phases.
+		if iface := vmispec.LookupInterfaceByNetwork(v.vmi.Spec.Domain.Devices.Interfaces, &networks[i]); iface.SRIOV != nil {
```

Comment:
It's not mandatory but it can be done for phase 1, I didn't change it now to not increase the scope of the PR further.
Can we do it on a follow-up PR?